frontend/packages/ceph-storage-plugin/locales/en/console-shared.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Troubleshoot": "Troubleshoot"
+}
```

Comment:
why this ?

Reply:
This might have popped up in a fixup commit. Removed.